### ISSUE TITLE
Email address is hard-coded in password reset template

### ISSUE THREAD

**Issue by saegey** [djangopackages-zz/djangopackages#90]
Email address is hard-coded in password reset template

**Comment by gisle** [comment]
You mean the "mailto:pydanny@djangopackages.com" in templates/account/password_reset_done.html?

**Comment by saegey** [comment]
Yes, I propose we move a lot of the site specific text blocks on the site that are not a static page to a 3rd party django app that supports "flat blocks" of text which are editable via the admin application
